Tax Loss Harvest Alert/README.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Tax Loss Harvest Alert with Google Finance
+description: List your stocks and purchase prices in a Google Sheet, and this script will alert you if you can sell the stock for a "tax loss harvest"
+labels: Apps Script, Sheets, Google Finance
+material_icon: event
+create_time: 2020-06-14
+update_time: 2020-006-14
+---
+
+Contributed by Jeremy Glassenberg [Site](https://www.apistrategist.com)
+
+When we purchase a stock and find the value dropped, there's an opportunity to sell that stock, purchase another, and claim a tax deduction.  However, many brokerages don't let you know when you have that opportunity.  
+
+This simple script works with Google Sheets to alert you for free.  Simply copy in your portfolio information, so the sheet knows what stocks you own and at what purchase prices. If anything falls below purchase price, you'll be notified of a tax loss harvest opportunity.
```

Comment:
Suggest rewording a little. We typically refrain from describing things as simple since it's vague (it might be simple for you, but not for somebody new to coding) unless it's truly trivial. Likewise, 'free' can also be problematic -- can be hidden or assumed 3P costs (e.g. ISP/bandwidth) or future changes in pricing that affect that.